refinery/templates/base.html

```diff
@@ -170,8 +181,13 @@
             </a>&nbsp;&nbsp;
           &nbsp;&nbsp;|&nbsp;&nbsp;
           {% endif %}
-          <i id="refinery-login" class="fa fa-sign-in"></i>&nbsp;<a rel="tooltip"
-          data-placement="left" title="Log In" href="{{login_url}}?next={% if request.path %}{% ifnotequal registration_url request.path %}{% ifnotequal login_url request.path %}{{ request.path|safe }}{% else %}{% url "home" %}{% endifnotequal %}{% else %}{% url "home" %}{% endifnotequal %}{% else %}{% url "home" %}{% endif %}">Login</a>
+        <a rel="tooltip" data-placement="left" title="Log In"
+        href="{{login_url}}?next={% if request.path %}{% ifnotequal registration_url request.path %}{% ifnotequal login_url request.path %}{{ request.path|safe }}{% else %}{% url "home" %}{% endifnotequal %}{% else %}{% url "home" %}{% endifnotequal %}{% else %}{% url "home" %}{% endif %}">
```

Comment:
This isn't new in your commit, but since all the `else`s have the same value, this could probably be rephrased as one if-then, with all the conditions and-ed together.